cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/Transactions.java

```diff
@@ -49,12 +49,26 @@ public static void abortQuietly(TransactionSystemClient txClient, Transaction tx
 
   public static void invalidateQuietly(TransactionSystemClient txClient, Transaction tx) {
     try {
-      txClient.invalidate(tx.getWritePointer());
```

Comment:
"quietly" means without throwing exceptions? Looks like this class is missing some javadocs for public methods